node-tests/blueprints/instance-initializer-test.js

```diff
@@ -116,6 +118,59 @@ describe('Blueprint: instance-initializer', function() {
     });
   });
 
+  describe.only('in app - module unification', function() {
+    beforeEach(function() {
+      return emberNew().then(() => fs.ensureDirSync('src'));
+    });
+
+    it('instance-initializer foo', function() {
+      return emberGenerateDestroy(['instance-initializer', 'foo'], _file => {
+        expect(_file('src/instance-initializers/foo.js')).to.contain(
+          'export function initialize(/* appInstance */) {\n' +
+          "  // appInstance.inject('route', 'foo', 'service:foo');\n" +
+          '}\n' +
+          '\n' +
+          'export default {\n' +
+          '  initialize\n' +
+          '};'
+        );
+
+        expect(_file('src/instance-initializers/foo-test.js')).to.contain(
+          "import { initialize } from 'my-app/instance-initializers/foo';"
```

Comment:
I think this will have to change to point to the new path under src

Reply:
Thanks @GavinJoyce, changed that--now trying to figure out how to get the tests actually passing!

When I have this worked out, I'll move onto the `addon` case, then update `instance-initializer-test-test`.